README.md

```diff
@@ -6,7 +6,8 @@ description: Language Server Protocol support with multiples languages support f
 [![](https://melpa.org/packages/lsp-mode-badge.svg)](https://melpa.org/#/lsp-mode)
 [![](https://stable.melpa.org/packages/lsp-mode-badge.svg)](https://stable.melpa.org/#/lsp-mode)
 [![](https://badges.gitter.im/emacs-lsp/lsp-mode.svg)](https://gitter.im/emacs-lsp/lsp-mode)
-[![](https://travis-ci.org/emacs-lsp/lsp-mode.svg?branch=master>)](https://travis-ci.org/emacs-lsp/lsp-mode)
+[![](https://github.com/ibizaman/lsp-mode/workflows/CI/badge.svg)](https://github.com/ibizaman/lsp-mode/actions?query=workflow%3ACI)
```

Comment:
The badge is pointing to your repo. This need to be fixed.
Also I think we can just remove the badge, Github already showing status anyway.
Like this
![image](https://user-images.githubusercontent.com/2631472/84562412-5ff45c80-ad8f-11ea-9d0b-c23c30232745.png)

Reply:
> Github already showing status anyway.

we might need it for https://emacs-lsp.github.io/lsp-mode/